app/konnect/org-management/audit-logging/reference.md

```diff
@@ -23,24 +49,17 @@ trace_id=3895213347334635099
 user_agent=grpc-go/1.51.0
 ```
 
-Each authentication log entry contains the following:
+Each authentication log entry contains the following attributes in addition to the defaults above:
```

Comment:
```suggestion
In addition to the defaults, each authentication log entry also contains the following attributes:
```
Avoid mentioning positioning on a page, since that can always change.